From 83ffb55cf27cfe0301f1c7ff4147832749e5917d Mon Sep 17 00:00:00 2001
From: Islam Amer <islam.amer@jollamobile.com>
Date: Wed, 22 Oct 2014 00:37:48 +0300
Subject: [PATCH] revert --hide-internal to work with vala 0.24.0

---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index b1ba0fd..6685b3a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -19,7 +19,7 @@ GLIB_REQUIRED=2.36
 VALA_ADD_CHECKFILE([gee/libgee_0_8_la_vala.stamp])
 VALA_ADD_CHECKFILE([tests/tests_vala.stamp])
 VALA_ADD_VALAFLAGS(--target-glib=$GLIB_REQUIRED)
-VALA_ADD_VALAFLAGS(--hide-internal)
+#VALA_ADD_VALAFLAGS(--hide-internal)
 
 AC_ARG_ENABLE(doc, AS_HELP_STRING([--enable-doc], [Enable documentation generation]), enable_doc=$enableval, enable_doc=no)
 AM_CONDITIONAL(ENABLE_DOC, test x$enable_doc = xyes)
@@ -66,7 +66,7 @@ AS_IF([test "x$enable_consistency_check" != xno], [VALA_ADD_VALAFLAGS(-D CONSIST
 
 GOBJECT_INTROSPECTION_CHECK([0.9.0])
 
-VALA_CHECK([0.25.1])
+VALA_CHECK([0.24.0])
 
 AC_OUTPUT([Makefile
            gee-0.8.pc
-- 
1.9.3

