From 52bff1fbca0e69a23c1c2ef3d43a1c0f36baab8c Mon Sep 17 00:00:00 2001
From: Islam Amer <islam.amer@jollamobile.com>
Date: Wed, 22 Oct 2014 02:34:39 +0300
Subject: [PATCH] fix broken sed

---
 gee/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gee/Makefile.am b/gee/Makefile.am
index 42e0a89..aa207fa 100644
--- a/gee/Makefile.am
+++ b/gee/Makefile.am
@@ -161,7 +161,7 @@ typelib_DATA = \
 # Extract dlname from libfolks.la; see bug #658002.
 # This is what g-ir-scanner does.
 libgee_dlname = \
-	`$(SED) -nE "s/^dlname='([A-Za-z0-9.+-]+)'/\1/p" libgee-0.8.la`
+        `$(SED) -n "s/^dlname='\(.*\)'/\1/p" libgee-0.8.la`
 INTROSPECTION_COMPILER_ARGS= \
 	-l $(libgee_dlname)
 Gee-0.8.gir: libgee_0_8_la_vala.stamp
-- 
1.9.3

