From 90e0042530e2aa895fc34cf120a5e87f525c7b4b Mon Sep 17 00:00:00 2001
From: Marko Saukko <marko.saukko@jolla.com>
Date: Wed, 29 Jan 2020 09:54:11 +0000
Subject: [PATCH] Our old sed does not support -E work around it.

Signed-off-by: Marko Saukko <marko.saukko@jolla.com>
---
 gee/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gee/Makefile.am b/gee/Makefile.am
index 42e0a89..af1b258 100644
--- a/gee/Makefile.am
+++ b/gee/Makefile.am
@@ -161,7 +161,7 @@ typelib_DATA = \
 # Extract dlname from libfolks.la; see bug #658002.
 # This is what g-ir-scanner does.
 libgee_dlname = \
-	`$(SED) -nE "s/^dlname='([A-Za-z0-9.+-]+)'/\1/p" libgee-0.8.la`
+	`$(SED) -n "s/^dlname='\(.*\)'/\1/p" libgee-0.8.la`
 INTROSPECTION_COMPILER_ARGS= \
 	-l $(libgee_dlname)
 Gee-0.8.gir: libgee_0_8_la_vala.stamp
-- 
1.8.3-rc3

