diff --git a/configure.ac b/configure.ac
index 7a14b92..5efb12a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -56,7 +56,10 @@ PKG_CHECK_MODULES(GLIB, glib-2.0 >= $GLIB_REQUIRED gobject-2.0 >= $GLIB_REQUIRED
 AC_SUBST(GLIB_CFLAGS)
 AC_SUBST(GLIB_LIBS)
 
+AM_CONDITIONAL(HAVE_INTROSPECTION, test "x$have_introspection" = xyes)
+m4_ifdef([GOBJECT_INTROSPECTION_CHECK], [
 GOBJECT_INTROSPECTION_CHECK([0.9.0])
+])
 
 AC_OUTPUT([Makefile
            gee-1.0.pc
